### PR TITLE
Update attendee prerequisites

### PR DESCRIPTION
**Pull request by atoato88**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR updates attendee prerequisites in japan folder.
This modified some sentences more precisely.

Signup process is chenged by EasyCLA, It needs to signup on the timing of workshop, not prerequisites.
It is more correct "サインアップすることを検討してください" not "サインアップしておいてください" in Japanese.
That's why I create this PR.

**Which issue(s) this PR fixes**:
NONE

**Special notes for your reviewer**:
NONE

**Does this PR introduce a user-facing change?**:
NONE

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
NONE
```

### CODE REVIEW

**Comment by atoato88** [comment]
/assign @t-inu 
Could you review this?
It's a little bit changes.

**Comment by atoato88** [comment]
/assign @s-kawamura-w664 
Oh, I should have assigned s-kawamura-w664, because previous PR #973 is lgtm by s-kawamura-w664.
Sorry for t-inu.
But it's also OK for reviewing by t-inu.

File: japan/assets/attendee-prerequisites.md [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 以下の [CNCF CLA サインアップ](#cncf-cla-へのサインアップ)において**企業の開発者として登録する**場合は、メールアドレスは企業ドメインのアドレス等を使用する必要があります。  
 詳しくは企業の[CLA Manager](https://docs.linuxfoundation.org/lfx/easycla/v2-current/corporate-cla-managers)に確認してください。  
-企業として参画する予定だが、ワークショップまでに間に合わない場合は、個人開発者としてサインアップしておいてください。
+企業として参画する予定だがワークショップまでに間に合わない場合は、個人開発者としてサインアップすることを検討してください。
```
Comment (oomichi) [inline japan/assets/attendee-prerequisites.md:18]: s/間に合わない/確認が間に合わない/

Comment (atoato88) [reply japan/assets/attendee-prerequisites.md:18]: I'll fix this :+1: +1

**Comment by oomichi** [comment]
Thanks for updating.

/lgtm

**Pull request merged**